scripts/deploy.sh

```diff
@@ -59,6 +59,6 @@ if [[ ${TRAVIS_SECURE_ENV_VARS} = true ]]; then
     done
 
     echo -e "\e[93m * Deploying app to google play (release version)...\e[0m"
-    ./scripts/basic_upload_apks_service_account.py ch.opengis.qfield ${ASSETS}
+    ./scripts/basic_upload_apks_service_account.py ch.opengis.qfield-dev ${ASSETS}
```

Comment:
@m-kuhn @mbernasocchi is this the channel we can push to?

Reply:
sounds good to me :+1:

Reply:
do you want to wait for Marco's review or we merge and see what happens?

Reply:
(still need an approval before being able to merge)